### DISCUSSION TITLE
Welcome to kraken Discussions!

### DISCUSSION THREAD

**Discussion by andycall** [openkraken/kraken#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChristianKleineidam** [comment]
It's unclear from the description in the readme what this project is about. Is it an alternative to Electron that runs also on Android/iOS? Or is it an alternative to a webview in Flutter?

**Reply by wssgcg1213** [reply]
It's more like a WebView, or a web standard(part of) implementation based on Flutter.

**Reply by andycall** [reply]
It is a Flutter-based web rendering engine, just like WebCore in Webview, which renders in the native way of Flutter instead of integrating an existing webview.
You use the JavaScript tech-stack (React, Vue) to build Flutter applications natively.

**Reply by ChristianKleineidam** [reply]
Why would I use it over just using a WebView to render my Vue app?

**Reply by andycall** [reply]
With no backwards compact of deprecated CSS features and more advanced newly CSS feature support. Kraken can achieve lower page load time and lower memory cost to provide better user experience on mobile or desktop apps.
